CSSS/Kernel.cs

```diff
@@ -157,6 +157,43 @@ public bool PerformTasks()
                 // so return 'true' from this function to let the
                 // calling main run loop know it can exit
                 shouldExit = true;
+
+                // If we should shutdown
+                if (config.CSSSProgramMode.HasFlag(Config.CSSSModes.Shutdown))
+                {
+                    // Run the shutdown command
+                    var process = new System.Diagnostics.Process();
+                    process.StartInfo.FileName = "shutdown";
+
+                    Console.WriteLine("Shutting down computer in 1 minute for image capture");
+
+                    logger.Info("Attempting to shut down the computer in 1 minute");
+                    switch (config.operatingSystemType)
+                    {
+                        case Config.OperatingSystemType.WinNT:
+                            logger.Info("Operating System detected as WinNT - shutting down computer in 5 seconds");
+                            // On Windows, we can specify the seconds to delay the shutdown
+                            // So we shutdown in 60 seconds 
+                            // The logic behind this is that shutting down the program immediately
+                            // may cause issues if there is a longer lasting process and allows
+                            // time for the program to gracefully exit without being terminated
+                            // as the computer shutsdown

```

Comment:
Could this be moved to above the `logger.Info` statement (before the `switch`), and read:

```
// The logic behind a minute delay before shutting down is to allow
// CSSS to finish running and gracefully exit, rather than being
// terminated as the computer shuts down
```